prophyc/tests/generators/test_python.py

```diff
@@ -27,9 +27,9 @@ def test_includes_rendering():
     ]
 
     ref = """\
-from foo import number_12, symbol_1
-from ni_knights import definition, pi, symbol_2
-from numbers import (
+from .foo import number_12, symbol_1
```

Comment:
It would be reasonable to add a testcase which would fail without the PR and starts passing with PR. Otherwise we have no test which ensures that what we add in this PR will continue to work properly.